.github/workflows/docker.yml

```diff
@@ -0,0 +1,51 @@
+name: Docker image
+
+on:
+  push:
+    branches: ["master"]
+
+env:
+  REGISTRY: ghcr.io
```

Comment:
Last i checked, we had been uploading go-waku images to docker-hub registry under statusteam. 
Here the images shall be uploaded to github's container registery.

@richard-ramos , @fryorcraken  Need your input here as to if we will be continuing to store docker images in dockerhub under status or use github. This would apply for all Waku images and not just go-waku as it makes sense to store all of them at one location.

Reply:
I would engage the discussion on our discord (infra channel) for this.